deployments/multiple-ragapps/README.md

```diff
@@ -21,7 +21,9 @@ docker compose up
 
 > _Note_: This will use the release images from from Docker Hub.
 
-The whole state of all services (RAGApps, Manager and Keycloak) will be persisted in the directory set by the `STATE_DIR` environment variable (defaults to `./data`).
+The whole state of all services (RAGApps, Manager and Keycloak) will be persisted in the directory set by the `STATE_DIR` environment variable (defaults to `${PWD}/data`).
+
+As `PWD` doesn't work on Windows, you'll need to rename the provided `.env.windows` file to `.env` and adjust the `STATE_DIR` variable to the absolute path of the `data` directory.
```

Comment:
@marcusschiesser my PowerShell on Windows 11 machine doesn't have issue with `${PWD` syntax, but i guess it will failed to older version (ex Windows 10). I think we can either guiding Windows user to set PWD instead of STATE_DIR, or don't use PWD in docker-compose and always required to specify STATE_DIR.